### ISSUE TITLE
Color contrast ratio of the focus indicator on "Radio" buttons with the background is 1.237:1 which is less than 3:1 in Dark mode: A11y_ML.NET Model Builder_Scenario_Non Text Contrast.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/machinelearning-modelbuilder#3072]
### GitHubTags
#A11ySev2; #A11yTCS;#DesktopApp; #Win32;#Model Builder - Machine Learning for .NET Developers;#BM-MlnetModelBuilder-Win32-OCt25;#Win32;#A11yMAS;#WCAG1.4.11;#Non Text Contrast;

### Environment Details:
Application Name: ML.NET Model Builder
Edition Windows 11 Enterprise
Version 25H2
OS build 26200.6899
Microsoft Visual Studio Enterprise
Channel: Canary
Version: 18.3.0 Insiders [11121.168.main]

### Repro Steps:

1. Launch visual studio.
2. Create a new .NET project or open an existing .NET project.
3. Right-click on the project in the solution explorer and Add > Machine Learning.
4. Tab till scenario and select a scenario.
5. Select training environment.
6. Tab till next step and press enter, tab till advanced data options and press enter.
7. Tab till Data formatting and press enter.
8. Verify that Color contrast ratio of the focus indicator on "Radio" buttons with the background is 1.259:1 which is less than 3:1 in Dark mode

### Actual Result:
Color contrast ratio of the focus indicator on "Radio" buttons with the background is 1.259:1 which is less than 3:1 in Dark mode

### Expected Result:
The color contrast ratio of the focus indicator on "Radio" button with the background should be greater than or equal to 3:1.

### User Impact:
When the color contrast is less than 3:1, it makes the text difficult to see. This can lead to poor visibility and hinder accessibility for users with visual impairments.

### Attachment:

<img width="962" height="453" alt="Image" src="https://github.com/user-attachments/assets/5f990a0e-51cb-4f49-885f-e7a36f68cf5a" />
